styles/utils/cssGrid.js

```diff
@@ -0,0 +1,95 @@
+import { rem } from 'styles/utils';
```

Comment:
this file already exists in `@irvingjs/styled`. that might change in the future, but we could reference it there for now.